numba/__init__.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import warnings
 
-from . import config, errors, runtests, types
+from . import config, errors, _runtests as runtests, types
```

Comment:
Note that this `_runtests as runtests` means `numba.runtests` will be different if you do
```python
import numba
print(numba.runtests)
```
or
```python
import numba
import numba.runtests
```
such that in the former case `numba.runtests` points to the `numba._runtests` module, but in the latter it points to the `numba.runtests` module, a subtle difference.
But that way we retain the `numba.runtests` and `numba.runtests.main` symbols.